### ISSUE TITLE
GraphQL Support

### ISSUE THREAD

**Issue by smbkr** [zed-industries/feedback#762]
### Check for existing issues

- [X] Completed

### Language

graphql

### Language server link

https://github.com/graphql/graphiql/tree/main/packages/graphql-language-service-server

**Comment by hovsater** [comment]
Not until there’s a plug-in API available.

**Comment by fbjork** [comment]
@hovsater ok, is there an issue we can track for that?

**Comment by hovsater** [comment]
Yeah, there’s https://github.com/zed-industries/community/issues/388 currently tracking that. 🙂

**Comment by XiNiHa** [comment]
I'd also like to have Relay LSP support (which is done by an extension in VSCode) which can be utilized by calling the Relay compiler binary with the `lsp` command ([ref](https://github.com/facebook/relay/blob/6e9e000642a4439532b85f96ee74f88072d585c4/vscode-extension/src/languageClient.ts#L23-L27))

**Comment by fbjork** [comment]
Why not add GraphQL support natively instead of plugins?

**Comment by JosephTLyons** [comment]
We will be going open source FairlySoon™️. After that, users will be able to propose languages we should bake into the editor, open PRs to do so, or help out with a plugin system so that we can have language extensibility!
